### ISSUE TITLE
Fix handling of old-style classes

### ISSUE THREAD

**Issue by chrivers** [chrivers/pyjaco#14]
Pyjaco should either correctly support old-style classes (and we should have unit tests to make sure we have the semantics in place), or (favorably), we should give an error if the user attempts to compile an old-style class.

**Comment by dusty-phillips** [comment]
I suggest a warning instead of an error.

**Comment by chrivers** [comment]
Yes, could be. But some kind of notice for sure. I'm not fully sure how different old- and new-style classes are, but I'd like to avoid subtle errors, where possible. So, if we warn the user but we know the code is going to break subtly, that _should_ be an error, I think. But first of all, I have to figure out why it doesn't do anything at the moment.
